compiler/testData/diagnostics/tests/multiplatform/headerClass/nestedClasses.kt

```diff
@@ -1,14 +1,22 @@
-// FIR_IDENTICAL
+import kotlin.reflect.KProperty
```

Comment:
This import actually don't belong to anly of test file. All content of file should be declared between `// FILE` directive an other file structure directive (`// MODULE` or `// FILE`) or end of file

Reply:
Thanks. It was somehow done by diagnostic test runs, but not sure how they worked before. 🤷